packages/terra-toggle-section-header/src/ToggleSectionHeader.jsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import SectionHeader from 'terra-section-header';
+import Toggle from 'terra-toggle';
+import styles from './ToggleSectionHeader.scss';
+
+const cx = classNames.bind(styles);
+
+const VALID_LEVELS = [1, 2, 3, 4, 5, 6];
+
+const propTypes = {
+  /**
+   * Content in the body of the toggle section header component that will be expanded or hidden.doc
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * Text to be displayed in the section header.
+   */
+  title: PropTypes.string.isRequired,
+  /**
+   * Sets the heading level. One of 1, 2, 3, 4, 5, or 6.
```

Comment:
I hate Oxford commas.

Reply:
Tell [`Heading.jsx`](https://github.com/cerner/terra-core/blob/master/packages/terra-heading/src/Heading.jsx#L15) ¯\_(ツ)_/¯

Reply:
![https://media.giphy.com/media/X1nICIXRE8JVu/giphy.gif](https://media.giphy.com/media/X1nICIXRE8JVu/giphy.gif)

Reply:
Is there a definition somewhere that indicates what each heading level renders as?

Reply:
imo, I think the description does enough to describe what gets rendered since it directly correlates to the html heading. If you wanted more on the description, perhaps you can add something like (ex: h1, h2, ...) to the comment?

Reply:
Yeah, lets change the comment to use h1, h2, h3 instead of 1, 2, 3...

Reply:
Done 87992e358eea05d7862dbd86bc033ae7c85f7e99